test/golomb-test.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const assert = require('bsert');
+const Block = require('../lib/primitives/block');
+const Opcode = require('../lib/script/opcode');
+const Script = require('../lib/script/script');
+const Golomb = require('../lib/golomb/golomb');
+const testVector = require('./data/filter-valid.json').slice(2);
+
+describe('Golomb test', function() {
+  it('Should match all valid input & output scripts provided as test to their respective filters', function() {
```

Comment:
Could you shorten the `it` statement to something more like:
`should match all valid input & output scripts` ?